dlt/destinations/impl/databricks/configuration.py

```diff
@@ -143,6 +143,9 @@ class DatabricksClientConfiguration(DestinationClientDwhWithStagingConfiguration
     keep_staged_files: Optional[bool] = True
     """Tells if to keep the files in internal (volume) stage"""
 
+    """Whether PRIMARY KEY or FOREIGN KEY constrains should be created"""
+    create_indexes: bool = False
```

Comment:
Question regarding naming. The implementation uses the term `constrains` in the code, which is imo true for PKs and FKs. But the flag is called `create_indexes`. Would it be a good idea to call the flag `create_constraints` to make the naming / terminology consistent?

Reply:
thanks for clarification @rudolfix
ok let's use create_indexes @christian-bay-backstage